html/changelogs/stuffstuffstuff.yml

```diff
@@ -0,0 +1,43 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes: 
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#################################
+
+# Your name.  
+author: LordFowl
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Cosmic Echoes can no longer move through phoron walls/windows."
```

Comment:
Tbh some of this changelog can be cut down to size.

Reply:
Such as?

Reply:
I think it doesn't need to be on 7 lines. Some of them can be moved into one line.

I suggest putting all the additions on one line, and all the tweaks of previous existing mechanics on one line, and all the bugfixes on another.

Reply:
Multi-line changelogs are preferable to changelogs with several line's worth of information condensed into a single paragraph - namely a changelog should be entries that are line-sized, not paragraph sized. Entries that convey little to no information are the ones that should be culled.

This PR changes a large number of things, so it is only natural that it should have a number of entries.